src/handlers/dashboard/get_chart_data.py

```diff
@@ -61,29 +62,33 @@ def _build_query(query_params: dict, filters: list, path_params: dict) -> str:
         select_str = f"{query_params['stratifier']}, {select_str}"
         group_str = f"{query_params['stratifier']}, {group_str}"
         columns.remove(query_params["stratifier"])
+        strat_str = f'AND {query_params["stratifier"]} IS NOT NULL '
```

Comment:
I'm not seeing any quoting of these query params? We should probably be escaping them, then adding double quotes (or verifying that they match `[a-zA-Z]+`), yeah? That's what the ruff warning `S608` is about, which is quieted below, yeah.

Does not need to block this PR, but feels important.

Reply:
yeah i was just thinking about how gross this approach to SQL construction was - I may make a ticket for this rather than deal with it as part of the PR

Reply:
#130